roles/uperf/tasks/start_server.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+- block:
+  ### <POD> kind
+  - include_tasks: init.yml
```

Comment:
We can get rid of init.yml file and include the tasks here

Reply:
There are options, and they should be fine too. I thought the init part although being used by start_server task only, but it is a little too specific and has churned more often at least while I was working on  this pr. That was the reason I separated it out.